jobspec/parse_service.go

```diff
@@ -858,6 +859,7 @@ func parseChecks(service *api.Service, checkObjs *ast.ObjectList) error {
 			"task",
 			"success_before_passing",
 			"failures_before_critical",
+			"on_update",
```

Comment:
Is this only for `-hcl1` flag backwards compatibility?

Reply:
yep thanks, removing